tests/test_samplings.py

```diff
@@ -322,7 +313,7 @@ def test_gaussian_pyfar():
 
     # test with single number of points
     c = samplings.gaussian(5)
-    isinstance(c, Coordinates)
+    isinstance(c, SamplingSphere)
```

Comment:
```suggestion
    assert isinstance(c, SamplingSphere)
```
otherwise nothing happens, hope it is not also True for Coordinates

Reply:
Thanks for catching. I found a few more occurances of this and corrected them all.